WalletWasabi/Tor/Socks5/Pool/Circuits/DefaultCircuit.cs

```diff
@@ -7,12 +8,14 @@ namespace WalletWasabi.Tor.Socks5.Pool.Circuits;
 /// Use this class for HTTP requests where privacy is not important really.
 /// This includes downloading fee rates from a 3rd party service, etc.
 /// </remarks>
-public class DefaultCircuit : ICircuit
+public class DefaultCircuit : INamedCircuit
 {
 	public static readonly DefaultCircuit Instance = new();
 
 	private static string RandomName = RandomString.CapitalAlphaNumeric(21);
 
+	private long _isolationId = 0;
```

Comment:
Is my thinking of this correct?
So when we create a new instance the Name will be random and the PW starts from zero. This way even if the pw starts from zero the combination of name and id will be different so it will end up in different Tor circuits.

Reply:
The default circuit is not using the user/pw (VersionMethodNoAuthRequired). Does this make any sense here?